composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "storj/uplink-php",
```

Comment:
`storj/uplink` should be a better fit for the name of the composer package. Perhaps, it does not make sense to have "php" in the name.

Reply:
Fixed